R/expr-is.R

```diff
@@ -31,8 +31,21 @@ is_while_expr <- function(pd) {
   pd$token[1L] == "WHILE"
 }
 
+
+#' Check whether `pd` is a function call
 #' @describeIn pd_is Checks whether `pd` is a function call.
-#' @keywords internal
+#' @family third-party style guide helpers
+#'
+#' @examples
+#' \donttest{
+#' code <- "x <- list(1:3)"
+#' pd <- styler:::compute_parse_data_nested(code)
```

Comment:
@lorenzwalthert Also, creating user-facing examples for these helpers functions requires using `compute_parse_data_nested()`, but `:::` is going to trip R CMD check somewhere. 

Currently, I am sidestepping this with `donttest`, but I'd highly recommend that we export `compute_parse_data_nested()` as well, since it's the workhorse that all helper functions rely upon. WDYT?

Reply:
Ok. We can export this as well. I don't see namespace autocompletion conflicts nor that this function would change its API in a breaking way soon.

Reply:
Thanks. I will make the necessary changes in this PR itself.